### PR TITLE
#9: handle lost TCP connections

### PR DESCRIPTION
**Pull request by elight**

### CODE REVIEW

File: lib/rack/amqp/server.rb [hunk 1]
```diff
@@ -31,36 +31,13 @@ def configuration
...
-            if type = headers['Content-Type']
-              amqp_headers[:content_type] = type
-            end
-            if enc = headers['Content-Encoding']
-              amqp_headers[:content_encoding] = enc
-            end
-
-            chan.direct("").publish(response, amqp_headers)
+          client.on_tcp_connection_loss do |connection, _|
+            connection.reconnect(false, 10)
```
Comment (jszmajda) [inline lib/rack/amqp/server.rb:35]: what are these magic args?

Comment (elight) [inline lib/rack/amqp/server.rb:35]: Hey, that's amqp's magic args, buster!

IIRC, the number is the number of seconds to wait before attempting to
reconnect. Don't recall about the Boolean.

Next step would be to make it configurable.

Comment (jszmajda) [inline lib/rack/amqp/server.rb:35]: we could just do inline assignments to keep it readable, or comments

``` ruby
connection.reconnect(whatever = false, reconnect_delay = 10)
```

Comment (elight) [inline lib/rack/amqp/server.rb:35]: Readable? Comments?? What have you done with Josh Szmajda????

**Pull request merged**